.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+before_install:
+  - npm install -g yarn@1.10.1
+install:
+  - yarn install --pure-lockfile
+script:
+  - yarn run lint
+  - yarn run test
```

Comment:
* Is it important which version of `yarn` is used? If not, I believe [travis will auto-detect that you have yarn if you have `yarn.lock` and runs `yarn` for you](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#using-yarn).
* Do we want to `cache: yarn`? https://docs.travis-ci.com/user/caching/#yarn-cache
* If you want this to have a coverage report on something like [coveralls](https://coveralls.io/), we can have
  ```js
  after_success:
    - yarn coverage
  ```
  where there is a `coverage` script that is piped to `coveralls`. You can then have a badge that is placed on the `README.md` to show the current coverage percent.
* I'm not yet familiar with the publish flow here yet, but should `lint` and `test` be part of an NPM hook like `prepublishOnly` and then this `script` run that hook (a safety precaution when trying to publish the package)?

Reply:
> Is it important which version of `yarn` is used? If not, I believe [travis will auto-detect that you have yarn if you have `yarn.lock` and runs `yarn` for you](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#using-yarn).

Didn't know this. Will take that out.

> Do we want to `cache: yarn`? https://docs.travis-ci.com/user/caching/#yarn-cache

🤔 My kneejerk response is no caching...

> If you want this to have a coverage report on something like coveralls, 

It's about six lines of code. I'm not worried about test coverage.

> * I'm not yet familiar with the publish flow here yet, but should `lint` and `test` be part of an NPM hook like `prepublishOnly` and then this `script` run that hook (a safety precaution when trying to publish the package)?

I don't know either. Will poke at stuffs... 🤔

Reply:
Travis doesn't seem to be doing anything anyway... Will take out & (maybe) try to add back in another scope.

Reply:
In the past we've fixed the `yarn` version because the latest version conflicted with `nyc`, making it always fail coverage detection.  If that's been resolved, then latest `yarn` is better.

Reply:
> 🤔 My kneejerk response is no caching...

Whatever you're comfortable with :+1:. I do it to save time on builds.

> It's about six lines of code. I'm not worried about test coverage.

:+1:. Here's why I mentioned it: you had asked about open source practices, and while you may not be worried about sharing the test coverage report, it can be a reassuring thing to others who are evaluating using a JavaScript tool.

> Travis doesn't seem to be doing anything anyway... Will take out & (maybe) try to add back in another scope.

Okay. Happy to lend a hand here in the future if an issue can be created for it specifying what you'd like to see.